### ISSUE TITLE
Why no training code?

### ISSUE THREAD

**Issue by EternityZY** [ioanacroi/unsup-learning-from-video#1]
Your job is very good and it is very helpful to my study. But why no training code?

**Comment by mslavescu** [comment]
I'm also intrrested in the training code, to use it for http://ossdc.org platform.

Would be great to have it ready to run in Google Colaboratory, like I provide these algorithms:
https://github.com/OSSDC/OSSDC-VisionBasedACC

Now with TPU support in Colab, training could be accelerated a lot, see:
https://twitter.com/fchollet/status/1047841218539925504

**Comment by ioanacroi** [comment]
Sorry for the late reply. We haven't provided the training code because it is pretty classic and the emphasize of our article is on how to obtain the model itself. Basically you only need a L2 loss added on top of the provided model and to feed the training data.
